x/data/simulation/genesis.go

```diff
@@ -0,0 +1,8 @@
+package simulation
+
+import "github.com/cosmos/cosmos-sdk/types/module"
+
+// RandomizedGenState generates a random GenesisState for the data module.
+func RandomizedGenState(simState *module.SimulationState) {
+	// TODO: #186
+}
```

Comment:
Updated the description in #186 to make sure we include this update when we add genesis import and export.